[2022-02-13 05:32:57,840] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-02-13T05:32:50.479450+00:00 [queued]>
[2022-02-13 05:32:58,379] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-02-13T05:32:50.479450+00:00 [queued]>
[2022-02-13 05:32:58,381] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-13 05:32:58,382] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-13 05:32:58,384] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-13 05:32:58,585] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-02-13 05:32:50.479450+00:00
[2022-02-13 05:32:58,679] {standard_task_runner.py:52} INFO - Started process 221 to run task
[2022-02-13 05:32:58,769] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'manual__2022-02-13T05:32:50.479450+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpz5wxwgjt', '--error-file', '/tmp/tmp5_1el868']
[2022-02-13 05:32:58,842] {standard_task_runner.py:77} INFO - Job 31: Subtask download_dataset_task
[2022-02-13 05:32:59,368] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-02-13T05:32:50.479450+00:00 [running]> on host b3b0a6764cf4
[2022-02-13 05:32:59,769] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-13T05:32:50.479450+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-13T05:32:50.479450+00:00
[2022-02-13 05:32:59,772] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-13 05:32:59,774] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-01.csv > /opt/***/yellow_tripdata_2021-01.csv']
[2022-02-13 05:32:59,806] {subprocess.py:85} INFO - Output:
[2022-02-13 05:34:59,775] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-13 05:34:59,813] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20220213T053250, start_date=20220213T053257, end_date=20220213T053459
[2022-02-13 05:34:59,866] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-13 05:34:59,909] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
